app/controllers/hyrax/admin/collection_type_participants_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# OVERRIDE FILE from Hryax v2.9.0
```

Comment:
- correct format
- fix spelling of Hyrax
- make sure override upgrade to Hyrax 3.4